[2021-03-10 07:02:08,066] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.storing_user 2021-01-16T00:00:00+00:00 [queued]>
[2021-03-10 07:02:08,071] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: data_processing.storing_user 2021-01-16T00:00:00+00:00 [queued]>
[2021-03-10 07:02:08,071] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 07:02:08,071] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-10 07:02:08,071] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-10 07:02:08,079] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): storing_user> on 2021-01-16T00:00:00+00:00
[2021-03-10 07:02:08,081] {standard_task_runner.py:51} INFO - Started process 42506 to run task
[2021-03-10 07:02:08,086] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'data_processing', 'storing_user', '2021-01-16T00:00:00+00:00', '--job-id', '104', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_processing.py', '--cfg-path', '/tmp/tmp3btu3868']
[2021-03-10 07:02:08,086] {standard_task_runner.py:76} INFO - Job 104: Subtask storing_user
[2021-03-10 07:02:08,218] {logging_mixin.py:103} INFO - Running <TaskInstance: data_processing.storing_user 2021-01-16T00:00:00+00:00 [running]> on host airflowvm
[2021-03-10 07:02:08,361] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_processing
AIRFLOW_CTX_TASK_ID=storing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-01-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-16T00:00:00+00:00
[2021-03-10 07:02:08,361] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-10 07:02:08,361] {bash.py:158} INFO - Running command: echo -e ".separator "," 
. import /tmp/users.csv userstwos" | sqlite3 /home/airflow/airflow/airflow.db
[2021-03-10 07:02:08,366] {bash.py:169} INFO - Output:
[2021-03-10 07:02:08,369] {bash.py:173} INFO - /tmp/users.csv:1: expected 3 columns but found 6 - extras ignored
[2021-03-10 07:02:08,373] {bash.py:177} INFO - Command exited with return code 0
[2021-03-10 07:02:08,384] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=data_processing, task_id=storing_user, execution_date=20210116T000000, start_date=20210310T070208, end_date=20210310T070208
[2021-03-10 07:02:08,395] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-10 07:02:08,428] {local_task_job.py:118} INFO - Task exited with return code 0
